io.github.spacingbat3.webcord.yml

```diff
@@ -10,12 +10,14 @@ finish-args:
   - --share=ipc
   - --socket=x11
   - --socket=pulseaudio
+  - --socket=system-bus
```

Comment:
This permission shouldn't be added, please remove it

Reply:
It gives an error without it, but sure.